### ISSUE TITLE
Issue with app and resource with the same name causing weird behaviour on the inspector

### ISSUE THREAD

**Issue by theocodes** [lotus/router#63]
Say we have an app called awards within a container arch mounted `at: '/awards'`
If we add `resources :awards` to routes
when running `lotus routes` the inspector does not output the right thing.

```
              awards GET, HEAD  /awards                        Awards::Controllers::Awards::Index
           new_award GET, HEAD  /awards/awards/new             Awards::Controllers::Awards::New
              awards POST       /awards                        Awards::Controllers::Awards::Create
               award GET, HEAD  /awards/awards/:id             Awards::Controllers::Awards::Show
          edit_award GET, HEAD  /awards/awards/:id/edit        Awards::Controllers::Awards::Edit
               award PATCH      /awards/awards/:id             Awards::Controllers::Awards::Update
               award DELETE     /awards/awards/:id             Awards::Controllers::Awards::Destroy
```

when it should be:

```
              awards GET, HEAD  /awards/awards                 Awards::Controllers::Awards::Index
           new_award GET, HEAD  /awards/awards/new             Awards::Controllers::Awards::New
              awards POST       /awards/awards                 Awards::Controllers::Awards::Create
               award GET, HEAD  /awards/awards/:id             Awards::Controllers::Awards::Show
          edit_award GET, HEAD  /awards/awards/:id/edit        Awards::Controllers::Awards::Edit
               award PATCH      /awards/awards/:id             Awards::Controllers::Awards::Update
               award DELETE     /awards/awards/:id             Awards::Controllers::Awards::Destroy
```

Here's a setup with the issue and a failing test.
https://github.com/theocodes/routertest

Note: Using master

Thanks

**Comment by theocodes** [comment]
I'm working on this.

**Comment by jodosha** [comment]
@theocodes I assume that https://github.com/lotus/utils/pull/82 fixed this problem. Would you love to write a test here that proves that the problem is solved? So we can be double sure that `PathPrefix` was the root cause of this issue.

Thank you very much! :dancer:

**Comment by AlfonsoUceda** [comment]
@jodosha he made this PR https://github.com/lotus/router/pull/64/files

**Comment by jodosha** [comment]
@AlfonsoUceda :sparkling_heart:

**Comment by jodosha** [comment]
Closed by #64
